test/tests/p5.Oscillator.js

```diff
@@ -1,87 +1,85 @@
-'use strict';
+var expect = chai.expect;
 
-  var expect = chai.expect;
+describe('p5.Oscillator', function () {
+  this.timeout(1000);
 
-  describe('p5.Oscillator', function () {
-    this.timeout(1000);
+  var osc = new p5.Oscillator();
+  var amp = new p5.Amplitude();
```

Comment:
I think these need to be instantiated inside of a 'before' lifecycle method (as described in my previous review)